### DISCUSSION TITLE
Add security.txt

### DISCUSSION THREAD

**Discussion by Arszilla** [f1multiviewer/issue-tracker#335] [Feature requests] [answered]
I recommend adding `security.txt` (https://securitytxt.org/) to `multiviewer.app`'s website, in order to allow responsible disclosures to take place (in an easy and effective manner) in case there are vulnerabilities within the application, regardless of the platform etc.

**Comment by f1multiviewer** [comment, marked as answer]
I've added this. Thanks for the suggestion :)
